client/gutenberg/editor/calypsoify-iframe.tsx

```diff
@@ -429,17 +425,14 @@ const mapStateToProps = ( state, { postId, postType, duplicatePostId }: Props )
 
 	const iframeUrl = addQueryArgs( queryArgs, siteAdminUrl );
 
-	// Prevents the iframe from loading using a cached frame nonce.
-	const shouldLoadIframe = ! isRequestingSites( state ) && ! isRequestingSite( state, siteId );
-
 	return {
 		allPostsUrl: getPostTypeAllPostsUrl( state, postType ),
 		currentRoute,
 		editedPostId: getEditorPostId( state ),
 		frameNonce,
 		iframeUrl,
 		postTypeTrashUrl,
-		shouldLoadIframe,
+		shouldLoadIframe: !! frameNonce,
```

Comment:
if we back up a step and store the request above…

```js
const frameNonce = requestFrameNonce( siteId );
```

…then we can send the two meaningful values here in a more explicit manner

```js
return {
	…
	frameNonce: frameNonce.data,
	…
	shouldLoadIframe: frameNonce.state === 'success',
	…
};
```

Collapsing to an empty string is also fine and meets the needs - just wanted to let you know this exists.

Also, why are we collapsing to an empty string vs. the default? Is that causing an error in the component if it's not a string type?